text-ui-test/runtest.sh

```diff
@@ -8,16 +8,30 @@ cd ..
 
 cd text-ui-test
 
+cp save_test.txt save.txt
+
 java  -jar $(find ../build/libs/ -mindepth 1 -print -quit) < input.txt > ACTUAL.TXT
 
 cp EXPECTED.TXT EXPECTED-UNIX.TXT
 dos2unix EXPECTED-UNIX.TXT ACTUAL.TXT
 diff EXPECTED-UNIX.TXT ACTUAL.TXT
 if [ $? -eq 0 ]
 then
-    echo "Test passed!"
-    exit 0
+    echo "Output: PASSED"
+    # compare save file to expected save file
+    diff save.txt save_result.txt
+    if [ $? -eq 0 ]
+    then
+        echo "Save file: PASSED"
+        echo "Test result: PASSED"
+        exit 0
+    else
+        echo "Save file: FAILED"
+        echo "Test result: FAILED"
+        exit 1
+    fi
 else
-    echo "Test failed!"
+    echo "Output: FAILED"
+    echo "Test result: FAILED"
     exit 1
 fi
```

Comment:
I think its best to leave the original code for text-ui-test, just change the `expected.txt` and `input.txt` files.